server/graphql/v2/object/TransferWise.ts

```diff
@@ -52,7 +51,7 @@ export const TransferWise = new GraphQLObjectType({
     requiredFields: {
       args: {
         currency: {
-          type: new GraphQLNonNull(TransferWiseCurrency),
+          type: new GraphQLNonNull(GraphQLString),
```

Comment:
What's the relationship with the PR?

Reply:
The expense flow was broken due to an update in TransferWise, I deprecated the type because I couldn't render the Expense edit page.